pom.xml

```diff
@@ -217,7 +215,6 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-javadoc-plugin</artifactId>
         <configuration>
-          <doclint>none</doclint>
```

Comment:
After fixing real javadoc problems, the doclint may be restored to its default value, which produces a lot of warnings, but does not fail the build. 
Reasonable alternative setting may perhaps be:
`<doclint>all,-missing</doclint>`